hummingbot/strategy/liquidity_mining/data_types.py

```diff
@@ -12,6 +20,9 @@ def __repr__(self):
 
 
 class Proposal:
+    """
+    What is this? name of the market, PriceSize for sale, PriceSize for buy
```

Comment:
This is an order proposal class for LiquidityMining strategy. `market` is a trading pair of a market, e.g. ETH-USDT, `buy` is a buy order proposal (consists of order price and order size), and likewise for `sell`.

Reply:
Resolved in commit https://github.com/CoinAlpha/hummingbot/pull/3440/commits/b35ab311e01d62f7b1b468e7d6fb855932b78fa4